typescript-rest/example/cloud_example.ts

```diff
@@ -4,7 +4,8 @@ import { options, connect } from '../index'
 let tokenEnv = process.env["TDENGINE_CLOUD_TOKEN"];
 
 // get TDengine cloud token form env variables
-let urlEnv = process.env["TDENGINE_CLOUD_URL"];
+// let urlEnv = process.env["TDENGINE_CLOUD_URL"];
+let urlEnv = "http://hostname:port";
```

Comment:
Use env or `http://localhost:6041` since `port` should be a invalid port number.

Reply:
@xleili  please use http://localhost:6041 here.